lib/bitmap-file-helper.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = exports = {};
+
+exports.initFile = (path, callback) => {
+  fs.readFile(path, (err, data) => {
+    if(err) return callback(err);
+    return callback(null, data);
+  });
+};
+
+exports.writeNew = (path, data) => {
+  return fs.writeFile(path, data, (err, data) => {
+    if(err) return err;
+    return data;
+  });
+};
+
+exports.eraseFile = (path) => {
+  return fs.writeFile(path, '', (err, data) => {
```

Comment:
While this does change the contents of the file there is a better command from fs.unlink that would have erased the file entirely. Leaving it as an empty string can make it seem like there is still a relevant file.